### PR TITLE
Adding a Vagrantfile for spinning up a development VM

### PR DESCRIPTION
**Pull request by ofirtzabari**
Vagrant is a tool for using virtual machines to share development environments.
Learn more about it at:
https://www.vagrantup.com/

Also making changes to the `.gitignore` file to ignore any data that Vagrant
might create.

Fix issue #1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Vagrant
+.vagrant
```
Comment (kobihk) [inline .gitignore:132]: Please add a new line in the end of the file

**Review by kobihk — CHANGES_REQUESTED** [review]
Please add in the description something like:
Fix issue #1

**Comment by kobihk** [comment]
@ofirtzabari Please add in the description of this PR something like:
`Fix issue #1`

**Review by RonTuretzky — CHANGES_REQUESTED** [review]
I think you need to do the steps that will allow local port forwarding so we can display it in our browser

**Review by kobihk — APPROVED** [review]
Please add in the description something like:
Fix issue #1

**Review by EliasAndry — DISMISSED** [review]

File: Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://vagrantcloud.com/search.
+  config.vm.box = "fedora/34-cloud-base"
+  config.vm.synced_folder ".", "/vagrant", type: "virtualbox"
+  config.vm.provider "virtualbox" do |vb|
+	vb.memory = "1024"
+  end
+  config.vm.network(
+  "forwarded_port", guest: 8000, host: 8000, host_ip: "127.0.0.1"
+  )
```
Comment (kobihk) [inline Vagrantfile:22]: Please add indentation something like:
```suggestion
  config.vm.network(
	"forwarded_port", guest: 8000, host: 8000, host_ip: "127.0.0.1"
  )
```

**Review by kobihk — CHANGES_REQUESTED** [review]

File: Pipfile [hunk 3]
```diff
@@ -0,0 +1,12 @@
+[[source]]
```
Comment (kobihk) [inline Pipfile:1]: IMHO the pipfiles should be part of PR #9

**Review by kobihk — CHANGES_REQUESTED** [review]

**Comment by kobihk** [comment]
@ofirtzabari AFAICS you deleted all files
Please add back the Vagrantfile and the .gitignore files

**Review by kobihk — APPROVED** [review]

**Pull request merged**